js/screens/game/AnswerButton.js

```diff
@@ -21,6 +21,7 @@ export default class AnswerButton extends React.Component {
   render() {
     return (
       <TouchableOpacity
+        disabled={this.props.disabled}
```

Comment:
Would it make sense to destructure the props above?

Reply:
Yes!